module-apps/apps-common/notifications/NotificationsListPresenter.cpp

```diff
@@ -77,7 +77,10 @@ auto NotificationsListPresenter::create(const notifications::NotSeenSMSNotificat
     -> NotificationListItem *
 {
     auto item = new NotificationWithEventCounter(notifications::NotificationType::NotSeenSms, notification->getValue());
-    setNotificationText(item, notification, "app_desktop_unread_messages");
+    setNotificationText(item,
+                        notification,
+                        (notification->getValue() == 1) ? "app_desktop_unread_single_message"
```

Comment:
what does this "1" mean?

Reply:
This means that there's only 1 notification :D I agree it's confusing, but it's more a matter of the too generic name `getValue()`